### DISCUSSION TITLE
Implement overlay (Solved)

### DISCUSSION THREAD

**Discussion by 01fiore** [mocaccinoOS/mocaccino#87] [General] [answered]
Can we implement an overlay under Mocaccino? merci.

**Comment by joostruis** [comment]
Please add some motivation about why you think we need an overlay.

**Reply by 01fiore** [reply]
To install my Brother printer drivers.

https://gpo.zugaina.org/media-gfx/brother-dcp1610w-bin

**Comment by joostruis** [comment, marked as answer]
We do not plan to maintain an overlay, there is no need for that. What you can do is request this in community repository: https://github.com/mocaccinoOS/community-repository/issues
